src/app/components/WorkDetail/WorkDetail.jsx

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import { Link } from 'react-router';
+import {
+  isEmpty as _isEmpty,
+} from 'underscore';
+import appConfig from '../../../../appConfig';
+
+const WorkDetail = (props) => {
+  if (!props.detail) {
+    return null;
+  }
+
+  const { item } = props.detail;
+  const ebookUrl = appConfig.ereader[process.env.APP_ENV];
+
+  const parseEntities = (entities) => {
+    return (
+      <ul>
+        {entities.map((entity, i) => (
+          <li key={i.toString()}>
+            <Link to={{ pathname: '/', query: { author: `${entity.name}` } }}>{entity.name}, {entity.role}</Link>
```

Comment:
This doesn't work and it'll be easier to make this work if the search component routes to `/?author=....` itself.